processor/src/test/java/org/mapstruct/ap/test/nestedbeans/mixed/AutomappingAndNestedTest.java

```diff
@@ -200,7 +204,7 @@ public void shouldAutomapAndHandleSourceAndTargetPropertyNestingAndExpresion() {
 
         assertThat( target.getQuality() ).isNotNull();
         assertThat( target.getQuality().getReport() ).isNotNull();
-        assertThat( target.getQuality().getReport().getVerdict() ).isEqualTo( "PASSED" );
```

Comment:
Hey @filiphr .. this is not right.. should be automapped.. 
quality --> quality automapped
report --> report automapped
verdict is a property both in report and reportdto. So Should be automated.